Footer

chelseymedina opened the pull request:
Added css to separate in columns. Still need to modify soon for valid info and hyperlinking to other pages.

chelseymedina committed:
Add Temporary Info

these shall be changed

chelseymedina committed:
Separate in columns

CyrusVatandoost commented:
Looks good! However how about make it into a table instead? It looks fine in full screen but when the screen get's smaller it messes up.

![image](https://user-images.githubusercontent.com/22030882/32991754-aabba7fa-cd7c-11e7-81c6-d61deef73ba7.png)

chelseymedina commented:
Okay I'm gonna work on that!
Update: Done, check again?

chelseymedina committed:
Footer in table form

CyrusVatandoost committed:
Deleted Close Tags

Close tags would automatically close once they reach another open tag (except <div> and others).

Also changed the css style of the footer.
